lsp-mode.el

```diff
@@ -5103,10 +5106,24 @@ It will show up only if current point has signature help."
       (funcall action-handler action)
     (lsp--send-execute-command command arguments?)))
 
-(lsp-defun lsp-execute-code-action ((action &as &CodeAction :command? :edit?))
+(defun lsp-execute-code-action (action)
   "Execute code action ACTION.
-If ACTION is not set it will be selected from `lsp-code-actions-at-point'."
+If ACTION is not set it will be selected from `lsp-code-actions-at-point'.
+Request codeAction/resolve for more info if server supports."
   (interactive (list (lsp--select-action (lsp-code-actions-at-point))))
+  (if (-> (lsp--server-capabilities)
```

Comment:
please use `lsp-feature?` for that(refer to `lsp-method-requirements`).

Reply:
I searched about it but for server capabilities, I could not find how to do that, do you know how I can check for the resolveSupport via that?

Reply:
Check how textDocument/prepareRename is handled in lsp-method-requirements.

Reply:
You'll need to use `(lsp-feature? "codeAction/resolve")` and to add `("codeAction/resolve" :capability :resolveSupport)` to `lsp-method-requirements`.

Reply:
Nice, I'll change it

Reply:
> You'll need to use `(lsp-feature? "codeAction/resolve")` and to add `("codeAction/resolve" :capability :resolveSupport)` to `lsp-method-requirements`.

resolveProvider is not top level, so a custom check-command will be needed.

Reply:
Done!